addons/sale_timesheet/tests/test_sale_service.py

```diff
@@ -400,9 +396,9 @@ def test_sale_task_in_project_with_project(self):
         })
 
         self.assertTrue(so_line1.task_id, "so_line1 should create a task as its product's service_tracking is set as 'task_in_project'")
-        self.assertEqual(so_line1.task_id.project_id, self.project_global, "The project on so_line1's task should be project_global as configured on its parent sale_order")
+        self.assertEqual(so_line1.task_id.project_id, so_line1.project_id, "The project on so_line1's task should be so_line's project as configured on its parent sale_order")
```

Comment:
It's not configured on the parent sales order anymore, is it? Does this test even make sense anymore, since it's related to the `project_id` on the Sales Order?